app/api/db/models/user_models.py

```diff
@@ -3,45 +3,44 @@
 from typing import Optional
 from uuid import UUID
 
-from sqlalchemy import Boolean, Column, Date, ForeignKey, Text
-from sqlalchemy.dialects.postgresql import UUID as postgresUUID
+from sqlalchemy import Boolean, Column, Date, Enum, ForeignKey, Text
+from sqlalchemy.dialects import postgresql
 from sqlalchemy.orm import Mapped, relationship
 
 import api.logging
 
-from .base import Base, TimestampMixin, uuid_gen
+from . import base
 
 logger = api.logging.get_logger(__name__)
 
 
-class RoleEnum(str, enum.Enum):
-    USER = "User"
-    ADMIN = "Admin"
-    THIRD_PARTY = "Third Party"
+class RoleType(str, enum.Enum):
+    USER = "USER"
+    ADMIN = "ADMIN"
 
 
-class User(Base, TimestampMixin):
+class User(base.Base, base.IdMixin, base.TimestampMixin):
     __tablename__ = "user"
 
-    user_id: Mapped[UUID] = Column(postgresUUID(as_uuid=True), primary_key=True, default=uuid_gen)
-
     first_name: str = Column(Text, nullable=False)
     middle_name: Optional[str] = Column(Text)
     last_name: str = Column(Text, nullable=False)
     phone_number: str = Column(Text, nullable=False)
     date_of_birth: date = Column(Date, nullable=False)
     is_active: bool = Column(Boolean, nullable=False)
 
-    roles: Optional[list["UserRole"]] = relationship(
-        "UserRole", uselist=True, back_populates="user"
-    )
+    roles: Optional[list["Role"]] = relationship("Role", back_populates="user")
 
 
-class UserRole(Base, TimestampMixin):
-    __tablename__ = "user_role"
+class Role(base.Base, base.TimestampMixin):
```

Comment:
Why change this to `base.Base` from `Base`?

Reply:
Not a strong reason. I think I was being flip floppy about conforming to the [google styleguide](https://google.github.io/styleguide/pyguide.html#22-imports) which was used on one of the tech challenges. The google styleguide says to only import modules not individual classes or functions.

Happy to switch it back though, since for a lot of these libraries, their documentation and examples explicitly suggest patterns that are different from the google styleguide.